docs/shared/guides/turbo-and-nx.md

```diff
@@ -173,6 +173,5 @@ Nx was released in 2016. Turborepo was open sourced in December of 2021. Turbore
 - There are about [3 million downloads per week](https://www.npmjs.com/package/@nx/tao).
 - There are about 1 million+ unique [Nx Console](https://marketplace.visualstudio.com/items?itemName=nrwl.angular-console) (a plugin for VSCode) installations.
 - There is a rich ecosystem of [third-party plugins.](https://nx.dev/community)
-- There are 5500+ active users in [the community slack.](https://nrwlcommunity.slack.com/join/shared_invite/zt-jba969hz-e5zm1Ou_jiOP1J2h8UTu0w#/shared-invite/email)
```

Comment:
I removed this as our numbers now at not very impressive given we just launched today (~150 on day one it looks like)